packages/flutter_tools/lib/src/build_system/targets/native_assets.dart

```diff
@@ -121,12 +120,13 @@ class NativeAssets extends Target {
         );
       case TargetPlatform.linux_arm64:
       case TargetPlatform.linux_x64:
+      case TargetPlatform.windows_x64:
```

Comment:
Why only x64?

Reply:
Sorry, I thought target support (https://github.com/flutter/flutter/issues/129807) had landed recently with the build output structure change, but apparently not yet.